controllers/userController.js

```diff
@@ -55,23 +52,18 @@ module.exports = {
       }
 
       const passwordCompare = await bcrypt.compare(password, user.password);
-      // console.log(passwordCompare);
       if (passwordCompare) {
-        const { id: user_id, username: foundedUsername } = user;
-
         let token = crypto
           .createHash("md5")
-          .update(foundedUsername)
+          .update(`${username}${password}${process.env.SECRETKEY}`)
           .digest("hex");
-
-        const exTime = new Date();
-
-        await Access_Token.create({
-          user_id,
-          access_token: token,
-          expiry: exTime,
+        const expiryTime = new Date();
```

Comment:
if I remember correctly, the assignment is saying
expiry time should be + 1 hour but you are just saving current date&time @akaashinno

Reply:
sir, in this we are using middleware "verifyToken" in which we are checking expiry time i.e., 1hr. If expiry time is less than 1hr then it will go to controller else it will show error message.